src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars--N"></div>
```

Comment:
the class here should be `stars__star` not the `stars--N` (it's the class for parent: `<div class="stars stars--N">`).
The `stars--N` is the name example where you have to replace `N` and depending on the `N` the number of active start should be shown
```suggestion
      <div class="stars__star"></div>
```